release_data_abc.py

```diff
@@ -2,6 +2,57 @@
 
 import abc
 
+UNUSED_STATUS = 0
+PENDING = 1
+FINISHED = 2
+FAILED = 3
+ABANDONED = 4
+
+
+class Release(object):
+  """Class for Release Object."""
+
+  def __init__(self, release_name):
+    """Initializes a release with a specified name."""
+
+    self.name = release_name
+    self.tasks = []
+    self.links = []
+    self.started = 0
+    self.labels = []
+    self.state = UNUSED_STATUS
+    self.last_modified = 0
+    self.last_active_task = ""
+
+  def from_dict(self, dict):
```

Comment:
This is just one level of conversion. What if a value is also a dict?

Reply:
based off of our current data model releases shouldn't have any values that are dicts, only single values and arrays